modules/BuildUtilities/src/server/PrepareConfigFiles.coffee

```diff
@@ -236,96 +161,76 @@ getApacheSpecificConfString = (config, apacheCompileOptions, acasHome) ->
 writeApacheConfFile = (config)->
 	acasHome = path.resolve(__dirname,acasHome)
 	apacheCompileOptions = getApacheCompileOptions()
-	if apacheCompileOptions != 'skip'
-		apacheSpecificConfString = getApacheSpecificConfString(config, apacheCompileOptions, acasHome)
-	else
-		apacheSpecificConfString = ''
+	apacheSpecificConfString = getApacheSpecificConfString(config, apacheCompileOptions, acasHome)
 	rapacheConfString = getRApacheSpecificConfString(config, apacheCompileOptions, acasHome)
 	rFilesWithRoute = getRFilesWithRoute()
 	rFileHandlerString = getRFileHandlerString(rFilesWithRoute, config, acasHome)
 	fs.writeFileSync "#{acasHome}/conf/compiled/apache.conf", [apacheSpecificConfString,rapacheConfString,rFileHandlerString].join('\n')
 	fs.writeFileSync "#{acasHome}/conf/compiled/rapache.conf", [rapacheConfString,rFileHandlerString].join('\n')
 
-csUtilities.getConfServiceVars sysEnv, (confVars) ->
```

Comment:
I guess we are removing support for an external config service here, which was likely only used at one now-defunct customer.

Reply:
Yeah. This is not used any longer.  I don't think its a use case we need any more so this was unused code and was actually breaking running prepare config files when customer specific server functions isn't available like in the context of racas and roo. I want to maintain the same script between all 3.